token/program-2022/src/extension/mod.rs

```diff
@@ -907,10 +910,12 @@ pub enum ExtensionType {
     MetadataPointer,
     /// Mint contains token-metadata
     TokenMetadata,
-    /// Test variable-length mint extension
     /// Mint contains a pointer to another account (or the same account) that holds group
     /// configurations
     GroupPointer,
+    /// Mint contains token group configurations
+    TokenGroup,
+    /// Test variable-length mint extension
```

Comment:
Oops, I totally missed that one in the last review :disappointed:

Reply:
Ngl I thought I rebased it to the previous commit lol.